resource-service/README.md

```diff
@@ -18,46 +18,85 @@ limitations under the License.
 
 # Resource Service
 
-The Resource service accepts an incoming message from the `user` Kafka topic which contains the resourceId that is being accessed 
-(this could be an actual file, or a directory that could contain many files and/or sub-directories). The service will then query the backing store to
-see if the requested resourceId has been stored. If this is not the case then the request will be passed onto the local implementation of 
-the Resource service. All the returned resources will be within an Akka stream, each element in the stream is then consumed and
-added to the `resource` Kafka topic to be processed by the Policy service.
+At it's core, the Resource Service is an implementation of the `ResourceService` interface found [here](https://github.com/gchq/Palisade-common/blob/develop/src/main/java/uk/gov/gchq/palisade/service/ResourceService.java). 
 
-## Message Model and Database Domain
+This implementation is then wrapped with various service layers to work with the Palisade streaming architecture. 
+These additional layers include:
+* Pulling requests from an upstream kafka topic and writing responses to a downstream topic, or an error topic if an exception was thrown -
+  see [AkkaRunnableGraph](src/main/java/uk/gov/gchq/palisade/service/resource/stream/config/AkkaRunnableGraph.java)
+* Calling the provided implementation of the `ResourceService` -
+  see [SimpleResourceService](src/main/java/uk/gov/gchq/palisade/service/resource/service/SimpleResourceService.java) or [ConfiguredHadoopResourceService](src/main/java/uk/gov/gchq/palisade/service/resource/service/ConfiguredHadoopResourceService.java)
+* 'Caching' resources returned by previous `ResourceService` calls -
+  see [ResourceServicePersistenceProxy](src/main/java/uk/gov/gchq/palisade/service/resource/service/ResourceServicePersistenceProxy.java) and [ReactivePersistenceLayer](src/main/java/uk/gov/gchq/palisade/service/resource/repository/ReactivePersistenceLayer.java)
+
+> :warning:
+One provided implementation of this `ResourceService` interface is the [ConfiguredHadoopResourceService](src/main/java/uk/gov/gchq/palisade/service/resource/service/ConfiguredHadoopResourceService.java). 
+Windows users will have problems with Hadoop integration testing. 
+Included [here](src/component-tests/resources/hadoop-3.2.1/bin) is a Windows-compatible set of hadoop binaries. 
+To 'install' on Windows, copy the [hadoop.dll](src/component-tests/resources/hadoop-3.2.1/bin/hadoop.dll) to `C:\Windows\System32`. 
+This should enable the `HadoopResourceServiceTest` to run correctly. 
```

Comment:
There are no hadoop tests in the resource service anymore, this is no longer the best place for this.